docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/LightGBMBinaryClassification.cs

```diff
@@ -1,8 +1,8 @@
 ﻿using Microsoft.ML.Transforms.Categorical;
 
-namespace Microsoft.ML.Samples.Dynamic
+namespace Microsoft.ML.Samples.Dynamic.Trainers.BinaryClassification
```

Comment:
i don't think it is necessary to organize by namespaces to this granularity.

Reply:
This change was suggested by @shmoradims  in an earlier version of this PR. Can we get an agreement on the namespaces?

Reply:
We have trainers with the same name under different catalogs (binary, multi, regression, etc). We either have to use namespaces or class names to differentiate them. E.g:

Microsoft.ML.Samples.Dynamic.LightGbmBinaryClassification vs
Microsoft.ML.Samples.Dynamic.Trainers.BinaryClassification.LightGbm

We also have LightGbm in multiclass and ranking.

I prefer the latter option because class names are identical to the extension method APIs, and namespaces mirror the catalog path.


---
In reply to: [260413531](https://github.com/dotnet/machinelearning/pull/2729#discussion_r260413531) [](ancestors = 260413531)